### DISCUSSION TITLE
Use GC problem

### DISCUSSION THREAD

**Discussion by loxiaopohaiol** [goharbor/harbor#16013] [Q&A] [answered]
Hi, I install the habor with the v2.3.4, I first delete a Registry on the harbor website, the page return this registiy is delete  d, and when I use the GC button on the Harbor page, the harbor page is return 'Garbage Collection Successful', but I found in the storage with the registry, the manifests is not deleted, and then cause the blobs is not deleted. So Is that normal?

**Comment by loxiaopohaiol** [comment]
And I found some log in the jobservice: "Garbage Collection parameters: [delete_untagged: false, dry_run: false, time_window: 2]" Is that mean if the manifests is create in 2 hours, when I delete the registry on the Harbor page the manifests is not deleted? How can I change the time window? If I delete the registry, I want to delete the manifests immediately.

**Comment by wy65701436** [comment, marked as answer]
yes, there is an time window for GC to avoid removing the uploading blobs in non-blocking GC. It's not recommended to modify it, but if you'd like to do it, just add GC_TIME_WINDOW_HOURS=0 to harbor core env, and restart harbor core.
